### DISCUSSION TITLE
6 wire pinout

### DISCUSSION THREAD

**Discussion by dklemm** [visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA#554] [Q&A]
Once I've found VCC and GND are the remaining pin always soldered in the same order?

VCC (wire #1) 
GDN (wire #2) 
DATA (wire #3) 
CLK (wire #4) 
CS (wire #5) 
?? (wire #6) : this is just connected to the display and the pump wire is NC?

I have model 54327 - PRE2021.

**Comment by visualapproach** [comment]
It depends. They come in that order from the pump/dsp usually but you can use any pins that you set in config on the ESP. But I recommend using pcb_v2b layout.
The 6th wire is sound.
